index.html

```diff
@@ -1,11 +1,43 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Lesson 1</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TripHouse_Site</title>
+    <link href="style.css" rel="stylesheet">
 </head>
 <body>
-<h1>Моя первая страница</h1>
-<a href="About/index.html">Об авторе</a>
+<div class="advantages" id="advantages">
+
+    <h2 class="advantages-header">What do we offer</h2>
+    <hr>
+    <div class="advantages-cards-wrapper">
```

Comment:
**Semantic recomendation**
It's better to use tags` <ul>` and `<li>` here, because this is a list of elements